spec/spec_helper_acceptance.rb

```diff
@@ -144,10 +162,21 @@ def teardown_puppet_on(host)
   step "Purge puppet from #{host}"
   # Note pc1_repo is specific to the module's manifests. This is knowledge we need to clean
   # the machine after each run.
+  case host['platform']
+  when /debian|ubuntu/
+    on host, '/opt/puppetlabs/bin/puppet module install puppetlabs-apt', { :acceptable_exit_codes => [0,1] }
```

Comment:
Yes, always because the migration module doesn't move modules. It probably should, but I'd file another ticket for that.

On puppet helpers, that was BKR-273 (Beaker paths aren't setup for AIO after the upgrade), and is fixed in another PR.